src/test/resources/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqBuildReferenceTaxonomy/test.dict

```diff
@@ -13,4 +13,4 @@
 @SQ	SN:ref|ACC_3c	LN:99	M5:b091f1dc6e3c6ef4391fabc5d8149f05	UR:file:/Users/markw/IdeaProjects/gatk/src/test/resources/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqBuildReferenceTaxonomy/test.fasta
```

Comment:
This is a fair point but it's inconsequential (maybe the UR's are used in production?), plus it seems like all dict files in the repo have the same issue.